dali/kernels/imgproc/structure/label_bbox.h

```diff
@@ -147,15 +148,113 @@ void GetLabelBoundingBoxes(span<Box<ndim, Coord>> boxes,
                            TensorSlice<const Label, remaining_dims> in,
                            ivec<simplified_ndim> dim_mapping,
                            Label background,
-                           i64vec<ndim> pos = {}) {
+                           i64vec<ndim> pos) {
   int64_t n = in.size[0];
   int d = dim_mapping[simplified_ndim - remaining_dims];
-  for (int64_t i = 0; i < n; i++) {
-    pos[d] = i;
+  for (int64_t i = 0, p = pos[d]; i < n; i++, p++) {
+    pos[d] = p;
     GetLabelBoundingBoxes(boxes, ranges, hits, in.slice(i), dim_mapping, background, pos);
   }
 }
 
+
+/**
+ * @brief Calculates a bounding box for each lablel in `in`
+ *
+ * @param boxes     output bounding boxes; labels whose box index is outside of valid range of
```

Comment:
Yes, but the function doesn't allocate it and will not crash if there are more boxes than elements in the span - it will simply discard out-of-range labels.